warp.sh

```diff
@@ -132,12 +132,16 @@ if [ -n "${refresh_token}" ]; then
 	wg_public_key=$(printf %s "${wg_private_key}" | wg pubkey)
 	reg="$(cfcurl --header 'Content-Type: application/json' -H "Authorization: Bearer ${token}" "${BASE_URL}/reg/${device_id}")"
 else
+	# Ask the user for the name to use in the registration
+	read -p "Enter the desired name for the configuration (default: WG-device-00): " config_name
```

Comment:
Could you please make this more automation-friendly? Instead of waiting for stdin, read this from a new flag?